### ISSUE TITLE
enlarging

### ISSUE THREAD

**Issue by Bettinalexow** [SilverDecisions/SilverDecisions#162]
I a missing the function to enlarge the picture of the tree. Numbers are really. Would be could such a function, (Bettina Lexow CEMBA Cohort XXIV)

**Comment by arturkuter** [comment]
This can be solved by usuing a scroll + crl button just like zoom page

**Comment by Bettinalexow** [comment]
I agree but I would only enlarge the tree not the whole page.would be good to have such a function.

**Comment by mwasiluk** [comment]
There are already some options to enlarge some parts of the tree, e.g.:
1. font size: settings -> general -> font size
2. settings -> edge  -> stroke width
3. [node size in Layout](https://github.com/SilverDecisions/SilverDecisions/wiki/3.-Actions-supported-by-application#layout)

Settings are described here:
https://github.com/SilverDecisions/SilverDecisions/wiki/4.-Application-settings#settings-panel

@bkamins Maybe we should consider adding a SVG scaling option to the Layout menu?

**Comment by bkamins** [comment]
@Bettinalexow I believe that we allow to change the size of all elements of the tree (see the comment of @mwasiluk). Please let me know if anything is missing and we can add it.

**Comment by Bettinalexow** [comment]
I discovered using the tool the different possibilities of changing the size, but nevertheless would it be useful to have in the main window a zoom in/out button for the tree.

**Comment by bkamins** [comment]
OK - thank you. We will try to work on it after release 1.0.

@mwasiluk - maybe the simplest solution would be if zoom in / zoom out buttons changed the options responsible for sizing (tree layout, font size, node size)? Would it work.
